specification/app/resource-manager/readme.md

```diff
@@ -129,6 +129,15 @@ input-file:
   - Microsoft.App/preview/2025-02-02-preview/SourceControls.json
   - Microsoft.App/preview/2025-02-02-preview/Subscriptions.json
   - Microsoft.App/preview/2025-02-02-preview/Usages.json
+directive:
+  - suppress: AvoidAdditionalProperties
+    from: CommonDefinitions.json
+    reason: |
+      Do not introduce breaking changes in GA services 
+  - suppress: TrackedExtensionResourcesAreNotAllowed
+    from: LogicAppsExtension.json
+    reason: |
+      Do not introduce breaking changes in GA services 
 ```
```

Comment:
Add a where clause

Reply:
What are these suppressions for? when the rest of the changes are just related to prettier?

Reply:
These suppressions are also for CI failed in https://github.com/Azure/azure-rest-api-specs/pull/33950/checks?check_run_id=40635363308
Yes, the rest are for Prettier change